vllm/envs.py

```diff
@@ -246,6 +247,8 @@ def get_default_config_root():
     lambda: int(os.getenv("VLLM_CONFIGURE_LOGGING", "1")),
     "VLLM_LOGGING_CONFIG_PATH":
     lambda: os.getenv("VLLM_LOGGING_CONFIG_PATH"),
+    "VLLM_LOGGING_INTERVAL_SEC":
+    lambda: float(os.getenv("VLLM_LOGGING_INTERVAL_SEC", "5")),
```

Comment:
I wonder if there's a different name that helps clarify which logs this is associated with since it's not all logs.

`VLLM_LOGGING_STAT_INTERVAL_SEC` ?